base/utils/DESCRIPTION

```diff
@@ -27,6 +27,8 @@ Imports:
     data.table,
     dplyr,
     getPass,
+    dplyr,
+    getPass,
```

Comment:
This is almost certainly a merge error -- PLEASE remove it. It's small, but I'm disproportionately sore about it because it undoes #2170, which was a redo of #2070, which was removed by  a merge error identical to this one and we didn't notice for a month.